src/common/FormTabs.tsx

```diff
@@ -70,12 +70,20 @@ export default function FormTabs({
 
   useEffect(() => {
     formatUiWidgets()
-    if (mode === 'edit' || mode === 'newVersion') {
+    if (mode === 'edit') {
       let updatedSchema = cloneDeep(selectedSchema)
-      // Name should be disabled
+      // Version and name should be disabled in edit mode
       updatedSchema.schema.properties.highLevelDetails.properties.name.readOnly = true
+      updatedSchema.schema.properties.highLevelDetails.properties.modelCardVersion.readOnly = true
       setUpdatedSelectedSchema(updatedSchema)
-    } else {
+    }
+    if (mode === 'newVersion') {
+      let updatedSchema = cloneDeep(selectedSchema)
```

Comment:
You can deduplicate the code by moving the following two calls outside of the if statement.  They'll remain the same in both edit and newVersion mode?

```
       let updatedSchema = cloneDeep(selectedSchema)
       updatedSchema.schema.properties.highLevelDetails.properties.name.readOnly = true
```

Reply:
We discussed that this wasn't possible out of band.